curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-and-quality-assurance-projects/stock-price-checker.english.md

```diff
@@ -33,7 +34,7 @@ tests:
     testString: ''
   - text: If I pass along 2 stocks, the return object will be an array with both stock's info. Instead of likes, it will display rel_likes(the difference between the likes on both stocks) on both.
     testString: ''
-  - text: 'A good way to receive current price is the following external API(replacing ''GOOG'' with your stock): https://finance.google.com/finance/info?q=NASDAQ%3aGOOG'
+  - text: 'A good way to receive current prices is through our stock price proxy (replacing ''goog'' with your stock symbol): https://repeated-alpaca.glitch.me/v1/stock/goog/quote'
```

Comment:
```suggestion
  - text: 'A good way to receive current prices is through our stock price proxy (replacing ''GOOG'' with your stock symbol): https://repeated-alpaca.glitch.me/v1/stock/GOOG/quote'
```
Just wondering if we should maybe capitalize the stock name to `GOOG`, following stock name conventions.

I did see that the app description uses lowercase though - 
> symbol = msft | goog | aapl | ...

Reply:
Thank you for catching that. Would be good to follow the stock name convention in the description here and in the Glitch example. I'll go ahead and make those changes now.